backend/src/main/java/com/crm/config/SecurityConfiguration.java

```diff
@@ -34,7 +34,9 @@
             "/webjars/**",
             // -- other
             "/auth**",
-            "/account**"
+            "/sign-up**",
+            "/account**",
```

Comment:
W nazwiązaniu do mojego poprzedniego komentarza trzeba pamiętać by to też zmienić

Reply:
done